R/step_growth_rate.R

```diff
@@ -32,13 +32,13 @@
 #' @importFrom epiprocess growth_rate
 #' @export
 #' @examples
-#' r <- epi_recipe(case_death_rate_subset) %>%
+#' r <- epi_recipe(covid_case_death_rates) %>%
 #'   step_growth_rate(case_rate, death_rate)
 #' r
 #'
 #' r %>%
-#'   prep(case_death_rate_subset) %>%
-#'   bake(case_death_rate_subset)
+#'   prep(covid_case_death_rates) %>%
+#'   bake(covid_case_death_rates)
```

Comment:
```suggestion
#'   bake(new_data = NULL)
```

(unrelated to the PR, but may as well while we're here.